File: C++/MemberOfParliament.cpp
#include "MemberOfParliament.h"

MemberOfParliament::MemberOfParliament(std::string name, std::string position, std::string party) {
    this->name = name;
    this->position = position;
    this->party = party;
}

std::string MemberOfParliament::toString() {
    return "Name: " + name + " Position: " + position + " Party: " + party;
}

File: Java/src/MemberOfParliament.java
 /*
Name:Shreyash Parsekar
Roll No:44
Batch-3
MCA-Part-1
*/
public class MemberOfParliament {
    private String name;
    private String position;
    private String party;

    public MemberOfParliament(String name, String position, String party) {
        this.name = name;
        this.position = position;
        this.party = party;
    }

    @Override
    public String toString() {
        return String.format("Name:"+name+" Position:"+position+" Party:"+party);
    }
}
